plugins/rhacm-backend/src/helpers/config.test.ts

```diff
@@ -0,0 +1,83 @@
+import { ConfigReader } from '@backstage/config';
+import { getHubClusterFromConfig } from './config';
+
+const CONFIG_FILE = {
+  kubernetes: {
+    clusterLocatorMethods: [
+      {
+        type: 'config',
+        clusters: [
+          {
+            name: 'cluster1',
+          },
+          {
+            name: 'cluster2',
+          },
+          {
+            name: 'cluster3',
+          },
+        ],
+      },
+    ],
+  },
+  rhacm: {
+    hub: 'cluster2',
+  },
+};
+
+describe('getHubClusterFromConfig', () => {
+  it('should get the correct hub cluster from multiple configured clusters', () => {
+    const config = new ConfigReader(CONFIG_FILE);
+
+    const result = getHubClusterFromConfig(config);
+
+    const expected = config
+      .getConfigArray('kubernetes.clusterLocatorMethods')[0]
+      .getConfigArray('clusters')[1];
```

Comment:
This should be a static value. The current test implementation is like comparing:

```
getHubClusterFromConfig(config) == getHubClusterFromConfig(config)
```

Reply:
Resolved in the latest commit.